docs/articles/new-expensify/travel/Manage-Travel-Member-Roles.md

```diff
@@ -21,8 +17,8 @@ To assign a role to a travel member,
   - **Travel Arranger**: Can book travel for themselves and for other workspace members. Arrangers can be set to arrange travel for everyone in the workspace or for specific individuals only.
   - **Company Admin**: Can book travel for themselves as well as any other workspace members. They can also access administrative features to:
     - Define travel policies
-    - Add users 
-    - Remove users 
+    - Add Users
```

Comment:
@CortneyOfstad @dangrous I was reviewing this page as part of https://github.com/Expensify/Expensify/issues/459234 and noticed that these are not indented correctly. the last parts of this need to be a sub bullet, not a main bullet. Can you please update this and confirm when it's published that it has been applied correctly? Thanks
![image](https://github.com/user-attachments/assets/51aaf199-74fd-4139-9143-0593593c13d3)